data/src/main/java/org/apache/iceberg/data/TableMigrationUtil.java

```diff
@@ -140,7 +179,7 @@ private static Metrics getAvroMerics(Path path,  Configuration conf) {
       return new Metrics(rowCount, null, null, null, null);
     } catch (UncheckedIOException e) {
       throw new RuntimeException("Unable to read Avro file: " +
-              path, e);
```

Comment:
Good change in style, but this is probably fine as a single line too

Reply:
updated